R/LSTconfidenceIntervals.R

```diff
@@ -0,0 +1,407 @@
+    return()
+  
+  treePlotContainer <- createJaspContainer(title = gettext("Tree Plot"))
+  jaspContainer[["containerTreePlot"]] <- treePlotContainer
+  treePlotContainer$dependOn(c("treePlot", "treePlotAdditionalInfo", "fixAxisTreePlot",
+                               "fixAxisLower", "fixAxisUpper"))
+  treePlotContainer[["treePlot"]] <- createJaspPlot(title = "", width = 480, height = 320) 
+  
+  if (options$fixAxisLower >= options$fixAxisUpper) {
+    treePlotContainer$setError(gettext("Make sure to specify a lower bound that is lower than the upper bound for the x-axis."))
+    return()
+  }
+  
+  
+  meanCI <- jaspContainer[["computedConfidenceIntervals"]][["object"]]
+  
+  p <- ggplot2::ggplot(meanCI,      
+                       ggplot2::aes(x = mean,
+                                    y = Repetition)) +
+    ggplot2::geom_point(ggplot2::aes(color = colorCI), size = 6, shape = 18) +
+    ggplot2::geom_errorbarh(ggplot2::aes(xmin = lower,
+                                         xmax = upper, 
+                                         color = colorCI)) +
+    ggplot2::scale_color_manual(values = c("A" = "black", "B" = "red")) +
+    ggplot2::geom_vline(data = data.frame(options$mu),
+                        ggplot2::aes(xintercept = options$mu),
+                        linetype = "dashed") +
+    ggplot2::scale_y_continuous(breaks= jaspGraphs::getPrettyAxisBreaks(1:options$nReps)) +
+    ggplot2::xlab(gettextf("Observed mean and %s%% CI", options$confidenceIntervalInterval * 100)) 
+  
+  if (options$fixAxisTreePlot) {
+    p <- p + ggplot2::scale_x_continuous(limits = c(options$fixAxisLower, options$fixAxisUpper))
+  }
+  
+  p <- p +
+    jaspGraphs::geom_rangeframe() + # add lines on the x-axis and y-axis
+    jaspGraphs::themeJaspRaw()      # add the JASP theme
+  
+  if(isTryError(p))
+    treePlotContainer[["treePlot"]]$setError(.extractErrorMessage(p))
+  else
+    treePlotContainer[["treePlot"]]$plotObject <- p
+  
+  return()
+}
+
+.plotDatasetsOLD <- function(jaspContainer, options) {
+  if (!is.null(jaspContainer[["containerRainCloudPlots"]]) || !options$dataPlot)
+    return()
+  
+  rainCloudPlotsContainer <- createJaspContainer(title = gettext("Data plots"))
+  # rainCloudPlotsContainer$position <- 3
+  jaspContainer[["containerRainCloudPlots"]] <- rainCloudPlotsContainer
+  rainCloudPlotsContainer$dependOn(c("dataPlotShowN", "dataPlot"))
+  
+  rainData <- data.frame(y = unlist(jaspContainer[["simulatedDatasets"]][["object"]]), 
+                         group = rep(1:options$nReps, each = options$n))
+  
+  singlePlot <- createJaspPlot(title = "", width = 480, height = 320)
+  rainCloudPlotsContainer[["rainCloudPlotAll"]] <- singlePlot
+  p <- try(jaspTTests::.descriptivesPlotsRainCloudFill(rainData, "y", "group", 
+                                                       yLabel = "Dependent", 
+                                                       xLabel = "Rep", 
```

Comment:
```suggestion
                                                       yLabel = gettext("Dependent"), 
                                                       xLabel = gettext("Rep"), 
```
also, shouldn't `Rep` be `Repetition` for consistency with the label of the other plot?